contracts/apps/AragonApp.sol

```diff
@@ -27,6 +28,10 @@ contract AragonApp is AppStorage, Autopetrified, VaultRecoverable, EVMScriptRunn
         _;
     }
 
+    function () external payable {
+        require(isDepositable());
```

Comment:
What's the reason behind adding a fallback by default to all apps?  IMO apps should define their own fallback function if they want to receive ETH. It may seem weird that depending on the gas value, a deposit can happen or not, but I'd say deposits and recovers should be used as the last resort and when a call is already forwarded to app, its own logic should be responsible for deciding on what to do. 

In case we decide to keep this, we should definitely check for data being empty and value being greater than 0.

Reply:
Good point. Now that `DepositableDelegateProxy`s also need their depositing functionality explicitly enabled, I agree that apps should define their own fallback if they want to receive ETH.